lib/worker.js

```diff
@@ -53,6 +54,21 @@ function timeUntilTimeout() {
     return (process.env.__OW_DEADLINE - Date.now()) || DEFAULT_METRIC_TIMEOUT_MS;
 }
 
+function setConsoleLogPrefix(prefix) {
```

Comment:
Move this into `utils`? Or `commons`? It feels misplaced here. The worker should not have to worry about defining that logic - just use it.